# This contains the message for the Wind
float32 w_ns     # North steady (m/s)
float32 w_es     # East steady (m/s)
float32 w_ds     # Down steady (m/s)
float32 w_xg     # X body gust (m/s)
float32 w_yg     # Y body gust (m/s)
float32 w_zg     # Z body gust (m/s)
